docs/facts/known_issues.rst

```diff
@@ -7,34 +7,128 @@ This is the list of issues which are not or will not be fixed (yet...).
 * Under GitLab CI/CD the coloration may not be shown.
 
 
+Python < 3.7
+^^^^^^^^^^^^
+
+    .. versionchanged:: 4.0.0
+
+As of version 4 we no `longer support <../installation/index.html#python-3-7-0>`_
+python prior to version 3.7.
+
+This means you actually are unable to run with any version below python 3.7
+as a number of build-in features are missing and first introduced in 3.7
+
+The error message you might experience can be:
+
+.. code-block::
+
+    `Fatal Error: type object 'datetime.datetime' has no attribute 'fromisoformat'`
+
+.. code-block::
+
+    `ModuleNotFoundError: No module named 'dataclasses'`
+
+This can typically happens if you are using Ubuntu 18.x or 19.x
+
 Ubuntu 20.04.1 LTS Focal
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. versionadded:: 3.2.0
 
-In Ubunto release 20.04 they have removed a package name
+In Ubuntu release 20.04 they have removed a package name
 :code:`libffi.so.6` and upgraded it with version :code:`libffi.so.7`
 
 This means PyFunceble will trow an error like:
 
-:code:`ImportError: libffi.so.6: cannot open shared object file: No such file or directory`
+.. code-block:: console
+
+    ImportError: libffi.so.6: cannot open shared object file: No such file or directory
 
 The fix for this issue is then rather simple, add a softlink between the
 versions with :code:`ln -s`
 
 The complete line in my case was:
 
-:code:`sudo ln -s /usr/lib/x86_64-linux-gnu/libffi.so.7 /usr/lib/x86_64-linux-gnu/libffi.so.6`
+.. code-block:: console
+
+    sudo ln -s /usr/lib/x86_64-linux-gnu/libffi.so.7 /usr/lib/x86_64-linux-gnu/libffi.so.6
+
+However, the right way to do this is by first locate where your
+:code:`libffi.so.7` is with find
 
-However, the right way to do this is by first locate where your's
-:code:`libffi.so.7` is by
+.. code-block:: console
 
-:code:`find /usr/lib/ -type f -iname 'libffi.so.*'` and then apply the
-softlink to :code:`libffi.so.7`
+    find /usr/lib/ -type f -iname 'libffi.so.*'
+    
+Then apply the softlink to :code:`libffi.so.7`
 
 
 Combination of :code:`-f`, :code:`-uf` and :code:`--adblock`
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-You can not combine the usage of :code:`-f`, :code:`-uf` and :code:`--adblock`
-at the same time
+You can not combine the usage of :code:`-f`, :code:`-uf` with :code:`--adblock`
+simultaneously.
+
+
+------
+
+.. sectionauthor:: @spirillen
+
+PyFunceble v 3.2.x
```

Comment:
Where was I as I reviewed this 🤔  `v3.2.x` all together, please 😄

Reply:
Yeah you should had been looking in https://github.com/spirillen/PyFunceble/pull/41 :smirk:  So it's new...